sample/migrations/0005_page.py

```diff
@@ -29,7 +29,6 @@ class Migration(migrations.Migration):
                     "book",
                     models.ForeignKey(
                         on_delete=django.db.models.deletion.CASCADE,
-                        related_name="pages",
```

Comment:
Not sure the reason we had to remove the related_name guessing it's the django doctor comment it can be ignored, Since it's more of a personal preference.